xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.js

```diff
@@ -1239,3 +1239,53 @@ function init() {
 // End XWiki augmentation.
 return XWiki;
 }(XWiki || {}));
+
+
+// Used for "link": "editor" property.
+// TODO: move the entire definition of the function in jquery when #livetablecallback
+// from macros.vm, where it is called, will also be moved.
+var editProperty = function(fullname, classname, fieldname, cb) {
+  require(['jquery'], function($) {
+    let doc = new XWiki.Document('LiveTableResults', 'XWiki');
+    let durl = doc.getURL('view', 'xpage=plain&page=' + fullname + '&classname=' + classname + '&fieldname='
+      + fieldname);
+    let editorModal = (`#editorModal`);
+    $('.xwiki-livetable').children().append(editorModal);
+    $('#propEditor').modal('show');
+    $.ajax({
+      url : durl,
+      complete : function(e) {
+        $('#propEditor').on('shown.bs.modal', function () {
+          $('#propEditorForm').html(e.responseText);
+        })
+        $('#propEditorSave').one('click', function() {
+          let value = getValue($('#propEditorForm'), fieldname);
+          let surl = doc.getURL('view', 'xpage=plain&page=' + fullname + '&classname=' + classname + '&fieldname='
+            + fieldname + '&value=' + value);
+          $.ajax({
+            url : surl,
+            success : function() {
+              cb(value);
+            },
+            error : function() {
+              alert("$services.localization.render('platform.livetable.propEditor.failed')");
```

Comment:
@evalica would know better about the UX but I don't think we use the alerts to display errors. We generally prefer to use the XWiki Notification widget: https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/XWikiNotifications/

Reply:
Ideally we should use the Notification widget. Check also for dependency. 
In case Notification widget will be used, provide error and success ("Field updated", "Field saved", etc.) messages.